docs/storage/flysystem.md

```diff
@@ -95,7 +95,7 @@ vich_uploader:
     mappings:
         product_image:
             uri_prefix: /images/products
-            upload_destination: product_image_fs
```

Comment:
This change is incompatible with OneupFlysystemBundle version 3.
We need ti provide a valid solution for both versions

Reply:
Does this do it?

Since v4 is the default now we can leave the v3 way but as a comment

Reply:
Yes